homeassistant/config_entries.py

```diff
@@ -137,6 +137,7 @@ class ExampleConfigFlow(config_entries.ConfigFlow):
 FLOWS = [
     'cast',
     'deconz',
+    'device_tracker.geofency',
```

Comment:
Needs component.

Reply:
Yep, will refactor after https://github.com/home-assistant/home-assistant/issues/17933 is merged.

Reply:
My bad, reading PRs in wrong order 👍

Reply:
No worries, I did this one first and then realized nothing worked because I needed a component first!